src/js/steps/letters/generateHtml.js

```diff
@@ -134,9 +138,9 @@ module.exports = function (options) {
         // removing all current content within the container, adding the
         // letters element to the DOM (and saving it to the data object), and
         // adding the loading gif in whilst the book loads.
-        $book.empty();
-        data.lettersElement = $lettersContainer.appendTo($book);
-        data.loading = data.loading.appendTo($book);
+        // $book.empty();
+        data.lettersElement = $lettersContainer.prependTo($book);
```

Comment:
Why's the changed to prependTo? Just curious on this one, not a criticism at all.